src/chainstate/stacks/mod.rs

```diff
@@ -42,7 +42,7 @@ use util::hash::HASH160_ENCODED_SIZE;
 use util::strings::StacksString;
 use util::secp256k1::MessageSignature;
 
-use address::AddressHashMode;
+pub use address::AddressHashMode;
```

Comment:
Why add this?  Why not just import `address::AddressHashMode` in whatever module(s) that try to import `AddressHashMode` from here?

Reply:
Change coming from #1203 - addressed with https://github.com/blockstack/blockstack-core/pull/1202/commits/9b789bed7ba7d13cf9d0b355ea0f94070ee3d1da

Reply:
Gotcha.  Thanks for taking care of it nevertheless :+1: